src/Home.js

```diff
@@ -0,0 +1,15 @@
+// import { Link } from 'react-router-dom';
```

Comment:
- [ ] Kindly please remove all lines of old commented pieces of code, so the project gets cleaner 💪

Reply:
done